setup.py

```diff
@@ -1,8 +1,83 @@
 import setuptools
 import os
+import subprocess
 from os import path
 import sys
 
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+build_dir = os.path.join(current_dir, 'deps_build')
+
+if not os.path.exists(build_dir):
+    os.mkdir(build_dir)
+
+os.chdir(build_dir)
+
+dep_install_path = ''
```

Comment:
Probably we should set a full path, like AWS_C_INSTALL usually is?
And probably have it be a subdir of the build directory, instead of the build dir itself?
```
dep_install_path = os.path.join(os.getcwd(), 'install')
```